contao/classes/Frontend.php

```diff
@@ -442,13 +442,9 @@ public static function addToUrl($strRequest, $blnIgnoreParams=false, $arrUnset=a
 		if (\Config::get('addLanguageToUrl'))
```

Comment:
No because the legacy properties are mapped: https://github.com/contao/core-bundle/blob/develop/contao/library/Contao/Config.php#L504